src/bin/batch_refunds.rs

```diff
@@ -1,11 +1,17 @@
 use lib::{
-    appconfig::connect_to_database_and_migrate, jobs::batch_refunds::batch_refunds_by_day,
+    appconfig::connect_to_database_and_migrate,
+    jobs::batch_refunds::batch_refunds_by_day,
     settings::get_settings,
+    telemetry::{init_sentry, init_tracing},
 };
 
 #[actix_web::main]
 async fn main() -> std::io::Result<()> {
     let settings = get_settings();
+
+    init_tracing("cjms-batch-refunds", &settings.log_level, std::io::stdout);
+    let _guard = init_sentry(&settings);
```

Comment:
```suggestion
   init_sentry(&settings);
```

Reply:
Actually, we can't do this. The purpose of the guard is to prevent the Sentry stuff from being dropped out of scope until the application exits.